### ISSUE TITLE
Make the gps-alert available to screen readers

### ISSUE THREAD

**Issue by Malvoz** [stefanocudini/leaflet-gps#24]
Making the `.gps-alert` a [_live region_](https://www.w3.org/TR/wai-aria/#dfn-live-region) is important as it conveys to screen readers that the content should be announced when added to the DOM, or otherwise has been updated. This is done by adding the [`status` role](https://www.w3.org/TR/wai-aria/#status) (a live region role) to the `<div>`, and accompanying ARIA attributes:

```html
<div class="gps-alert" style="display: block;"
  role="status"
  aria-live="polite"
  aria-atomic="true">
  Geolocation error: User denied Geolocation.
</div>
```

The `role="status"` ARIA attribute is implicitly [`aria-live="polite"`](https://www.w3.org/TR/wai-aria/#aria-live) and [`aria-atomic="true"`](https://www.w3.org/TR/wai-aria/#aria-atomic), however it's best to be explicit with these values per [MDN advice](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#Preferring_specialized_live_region_roles):

> To maximize compatibility, add a redundant `aria-live="polite"` when using this role.

And looking at the [support table for `role=status`](https://a11ysupport.io/tech/aria/status_role#expectations), we can see that not all browsers convey the implicit `aria-atomic` value of `true` by announcing the entire region, so we're explicit about that too.

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/SlavaUkraini/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
